common-app-lib/src/cmr/common_app/humanizer.clj

```diff
@@ -27,16 +27,16 @@
   "Map of humanizer JSON type values to functions which take a field value and
   humanizer configuration and return a transformed field value. The functions
   can assume that the humanizer should be applied to the value."
-  (fn [humanizer value]
+  (fn [humanizer _value]
```

Comment:
I have been converting the defmulti/defmethods into a function with case statement.  We don't need polymorphism here especially since all of the to-human functions are in this namespace.  Polymorphism is a little slower with regard to performance of the software.
But I will let you decide if you want to keep this as is or convert it.  I am not going to comment on the defmethod functions below.

Reply:
Yes, I competely agree!